components/User.tsx

```diff
@@ -31,8 +34,33 @@ export default function User() {
                 <div className="my-4">
                     <p className="font-bold">Welcome to Nasa Photo viewer</p>
                 </div>
+                <div className="flex justify-between mx-2 py-2 my-2">
+                    <div onClick={async()=>{
+                        setLoading(true);
```

Comment:
Can we put the function inside of the onClick as a separate function?
```suggestion
                        setLoading(true);
```

Reply:
done